src/data/ideas.js

```diff
@@ -0,0 +1,19 @@
+const ideasUrl =
+    'https://fund-raising-project-server-side.azurewebsites.net/ideas';
+
+async function getIdeas() {
+    try {
+        const response = await fetch(ideasUrl);
+        if (!response.ok) {
+            throw new Error('Failed to fetch data');
+        }
+        return await response.json();
```

Comment:
Why do you need `await` here?

Reply:
Async\await is a bit complicated for me, I'll havev to read some more about it. Fixed.